Struct.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+var schema = `CREATE TABLE hospital
+(
+    id                 SERIAL PRIMARY KEY,
+    name               VARCHAR(100),
+    max_patient_amount INT
+);
+
+CREATE TABLE location
+(
+    id          SERIAL PRIMARY KEY,
+    name        VARCHAR(100),
+    hospital_id INT,
+    FOREIGN KEY (hospital_id) REFERENCES hospital (id)
+);
+
+CREATE TABLE patient
+(
+    id             SERIAL PRIMARY KEY,
+    name           VARCHAR(100),
+    illness        VARCHAR(200),
+    birth_date     DATE,
+    location_id    INT NOT NULL,
+    FOREIGN KEY (location_id) REFERENCES location (id)
+);`
+//structures
+type hospital struct {
+	HospitalID       int    `db:"id" json:"id"`
+	Name             string `db:"name" json:"name"`
+	MaxPatientAmount int    `db:"max_patient_amount" json:"Max patient amount"`

```

Comment:
The json representation should be `camelCase`. Do this for all the attributes for each model.
```suggestion
	MaxPatientAmount int    `db:"max_patient_amount" json:"maxPatientAmount"`
```